src/routes/game/components/elements/endGameScreen/EndGameScreen.tsx

```diff
@@ -35,17 +37,26 @@ const EndGameScreen = () => {
     playerID === 2 ? setPlayerID(1) : setPlayerID(2);
   };
 
+  const toggleShowStats = () => {
+    setShowStats(!showStats);
+  }
+
   return (
-    <div className={styles.cardListBox}>
+    <div className={`${styles.cardListBox} ${!showStats ? styles.reduced : ''}`}>
```

Comment:
Not important here but I use the class names library to put together conditionals for class names

Reply:
Sure, makes sense. Added another Commit to update it. I didn't use Github for some time, does it squash the commits on PR or shall I do it manually?

Reply:
No need to squash.